### PR TITLE
CP-15398: Make version, xsupdate and xensearch rebrandable

### PR DESCRIPTION
**Pull request by cheng--zhang**
Signed-off-by: Cheng Zhang <cheng.zhang@citrix.com>

### CODE REVIEW

File: Branding/branding.sh [hunk 1]
```diff
@@ -30,3 +30,7 @@ else
 fi
 
 BRANDING_XC_PRODUCT_VERSION=${BRANDING_PRODUCT_MAJOR_VERSION}.${BRANDING_PRODUCT_MINOR_VERION}.${BRANDING_PRODUCT_MICRO_VERSION}
+
+BRANDING_XC_PRODUCT_5_6_VERSION=5.6
+BRANDING_XC_PRODUCT_6_2_VERSION=6.2
+BRANDING_XC_PRODUCT_6_5_VERSION=5.0
```
Comment (GaborApatiNagy) [inline Branding/branding.sh]: It looks like there is typo here (6.5)

Comment (cheng--zhang) [inline Branding/branding.sh]: O, yes. I will modify it.
Thanks. @GaborApatiNagy

**Comment by cheng--zhang** [comment]
Hi @GaborApatiNagy  and @MihaelaStoica 
I have update the branch and fix the typo.
Please help to review it again.
Thanks.

**Comment by cheng--zhang** [comment]
Hi @MihaelaStoica 
I have add the last commit to use condition install for XenCenter.
Verified both Citrix and non-citrix version.

Please help to review and merge if possible.
Thanks.

**Comment by MihaelaStoica** [comment]
Looks good. Merging

**Pull request merged**